### PR TITLE
create room for more vendors

### PR DESCRIPTION
**Pull request by bewest**
This change puts in a placeholder for each vendor we've discussed.
Consider this a "strawman" for exploring what it looks like to dissect
the Nightscout experiences by vendor type.

I hope this provides some structure for additional comments and maybe a way keep up momentum.
@psonnera and I have traded some ideas about this, so hopefully this helps a bit.
There's no link into this new "vendors" area yet, but if you pull down the branch and serve or build it yourself, you can preview what is intended to be a fairly simply outline:detail organization for each vendor.

### CODE REVIEW

File: docs/vendors/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### Comparison
+
+|              | Heroku        | Digital Ocean  | Azure         | T1Pal        |
+| ------------ | ------------- | -------------- | ------------- | ------------ |
+| Install Day 1| DIY           | DIY            | DIY           | Managed      |
+| Maintenance Day 2 | DIY       | DIY            | DIY           | Automatic    |
+| Cost         | Free, sales emails | $5USD/month | Depends       | $11.99USD/month |
+| Support      | DIY / Facebook| DIY / Facebook | DIY / Facebook | Private support |
+| Community Contributions |  None | None        | Research use | contributors  |
```
Comment (StephenBrown2) [inline docs/vendors/index.md]: Ran this through [table-magic](https://stevecat.net/table-magic) to get even columns:
```suggestion
|                         | Heroku             | Digital Ocean  | Azure          | T1Pal              |
|-------------------------|--------------------|----------------|----------------|--------------------|
| Install Day 1           | DIY                | DIY            | DIY            | Managed            |
| Maintenance Day 2       | DIY                | DIY            | DIY            | Automatic          |
| Cost                    | Free, sales emails | $5/month (USD) | Depends        | $11.99/month (USD) |
| Support                 | DIY / Facebook     | DIY / Facebook | DIY / Facebook | Private support    |
| Community Contributions | None               | None           | Research use   | contributors       |
|                         |                    |                |                |                    |
```

Comment (StephenBrown2) [reply docs/vendors/index.md]: Also a thought, might want to invert the columns and rows if the providers will be added to, but the criteria will stay static.

Comment (psonnera) [reply docs/vendors/index.md]: @bewest good idea, but how do we handle MongoDB Atlas that's paired with Heroku in classic DIY, and can be used with Azure, but isn't involved with DO and others... Maybe an extra line for Database?

Comment (bewest) [reply docs/vendors/index.md]: I think the criteria is not quite a fixed set, yet.  This table only reflects web-hosting, not database hosting.  Nightscout requires both web host and database host.  For example, DigitalOcean, people can make a choice: do they want built-in mongo or an external mongo.  We will likely document both options.  Should we add a criteria for database: self-contained vs external provider vs optional?

Comment (StephenBrown2) [reply docs/vendors/index.md]: >  Should we add a criteria for database: self-contained vs external provider vs optional?

I think yes.

Comment (psonnera) [reply docs/vendors/index.md]: `Components, costs and complexity will depend on the solution you want to implement.

Some solutions below:



|                              | Heroku              | Digital Ocean           | MVPS                    | Azure               | T1Pal             |
| ---------------------------- | ------------------- | ----------------------- | ----------------------- | ------------------- | ----------------- |
| Install                      | DIY                 | DIY                     | DIY                     | DIY                 | Managed           |
| Maintenance                  | DIY                 | DIY                     | DIY                     | DIY                 | Automatic         |
| Cost (USD)                   | Free                | $5/month                | $3/month                | Depends             | $11.99/month      |
| Support                      | DIY / Facebook      | DIY / Facebook          | DIY / Facebook          | DIY / Facebook      | Private support   |
| Community Contributions      | None                | None                    | None                    | Research use        | Contributors      |
| Mongo database<br />External | None<br />Mandatory | Contained<br />Optional | Contained<br />Optional | None<br />Mandatory | Contained<br />No |
| Complexity                   | Medium              | High                    | High                    | High                | Low               |`

**Comment by bewest** [comment]
Now looks something like this. Added the phrase "self-serve" or "self-service".  Also broke out "web hosting" vs "database hosting" so we can include everything.  Feedback welcome.


|                              | Heroku              | Digital Ocean           | MVPS                    | Azure               | T1Pal             | MongoDB Atlas           |
| ---------------------------- | ------------------- | ----------------------- | ----------------------- | ------------------- | ----------------- | -----------------       |
| Install                      | DIY                 | DIY                     | DIY                     | DIY                 | Automatic         | DIY                     |
| Maintenance                  | DIY                 | DIY                     | DIY                     | DIY                 | Automatic         | DIY / self serve        |
| Cost (USD)                   | Free - $7/month     | $5/month                | $3/month                | Depends             | $11.99/month      | Free sandbox - $9/month |
| Support                      | DIY / Facebook      | DIY / Facebook          | DIY / Facebook          | DIY / Facebook      | Private support   | DIY                     |
| Community Contributions      | None                | None                    | None                    | Research use        | Contributors      | MongoDB                 |
| Web Hosting                  | Yes                 | Yes                     | Optional                | Yes                 | Yes               | No                      |
| Database Hosting             | No                  | Optional                | Optional                | Optional            | Yes               | Yes                     |
| Complexity                   | Medium              | Medium - High           | High                    | High                | Low               | High                    |

**Comment by psonnera** [comment]
At this point might be better to rotate it as vendors will outnumber features. Also this is too technical. I'll deploy the vendors page with limited linking until it gets populated. Sorry for the picture but now it's the easiest for me to give an idea.
image

**Comment by bewest** [comment]
@psonnera this is looking great.  One note, T1Pal is $11.99 USD, not $12.99.  See attached.

**Comment by psonnera** [comment]
@bewest fixed in the repo https://github.com/nightscout/nightscout.github.io/blob/source/docs/vendors/index.md but I'm stuck at gh-deploy after updating my Github token.

**Comment by psonnera** [comment]
@bewest pushed some contents for T1Pal https://github.com/nightscout/nightscout.github.io/commit/456221782caf781794f30d13178c2554c55069b5

**Pull request merged**